### PR TITLE
Preparation for release v2023.1 (CLI)

### PR DESCRIPTION
**Pull request by matsduf**
## Purpose

Preparation for release.

## Changes

Updates files `Changes`, `Makefile.PL` and `CLI.pm`.

## How to test this PR

Review the changes.

### CODE REVIEW

**Review by marc-vanderwal — APPROVED** [review]

File: Changes [hunk 1]
```diff
@@ -1,5 +1,21 @@
 Release history for Zonemaster component Zonemaster-CLI 
 
+v5.1.0 2023-06-21 (public release version)
+
```
[the remark this answers is not in the document]
Comment (pnax) [reply Changes:4]: #306 is also missing

[the remark this answers is not in the document]
Comment (matsduf) [reply Changes:4]: Some time ago it was decided not to include details just related to unit testing etc.

[the remark this answers is not in the document]
Comment (matsduf) [reply Changes:4]: I will update with #306 too.

**Review by marc-vanderwal — CHANGES_REQUESTED** [review]

File: lib/Zonemaster/CLI.pm [hunk 2]
```diff
@@ -11,7 +11,7 @@ use 5.014002;
 use strict;
 use warnings;
 
-use version; our $VERSION = version->declare( "v5.0.2" );
+use version; our $VERSION = version->declare( "v5.1.0" );
```
[the remark this answers is not in the document]
Comment (matsduf) [reply lib/Zonemaster/CLI.pm]: Corrected. I missed to change there.

**Review by pnax — APPROVED** [review]

**Pull request merged**